components/link/index.css

```diff
@@ -17,9 +17,10 @@ governing permissions and limitations under the License.
   background-color: transparent;
   /* Remove gaps in links underline in iOS 8+ and Safari 8+. */
   -webkit-text-decoration-skip: objects;
-  text-decoration: none;
+  text-decoration: underline;
   transition: color var(--spectrum-global-animation-duration-100) ease-in-out;
   outline: none;
+  cursor: pointer;
```

Comment:
Hmm, I don't think this is necessary for `<a>`, but perhaps they're using the styles elsewhere, so it's fine.